### ISSUE TITLE
Flipper gap on pool table

### ISSUE THREAD

**Issue by easyaspi314** [dozingcat/Vector-Pinball#164]
From some quick testing it seems like the flipper gap on the pool table might be a little too much, resulting in a lot of unavoidable straight down the middle deaths. I already had 4-5 of them in just a few minutes of testing.

**Comment by ecodix** [comment]
Yes, the design of this table is awesome, but keeping he ball on the field with best play is too random. And i think it's even more obvious in the outer lanes, wihch are too wide open. As there is no outer lane saver/blocker making a high score with best play is too random. Only weak point :)

**Comment by AlbeyAmakiir** [comment]
I have just had a game where the first ball went down the middle, and the second ball went down the left outer lane, both without getting close enough to a flipper for me to interact with the ball at all. Not very fun. 😅

I don't know enough about pool table design to suggest anything, though.

**Comment by AlbeyAmakiir** [comment]
Also... is it just me or does the ball move even faster on this table than others? Or maybe gravity is a little different? Something about the ball's movement feels off if I play any other table before this one.

**Comment by ecodix** [comment]
Also, it is so hard to take down the 4 sucessive drop targets and hit the red one behind : indeed all targets are reset at each new ball start. I played over one hundred games and never managed to hit the red one, despite i was only focused on this and a pretty good player.
I like the idea, though, would it be possible not to reset the drop targets on new ball, except if the last red one was hit ?

**Comment by jackmorgan8** [comment]
Hi folks,

I ported this table over to Vector Pinball, it's a fairly accurate replication of Bally's Eight Ball Deluxe with some rule changes and added elements. 
It's supposed to be difficult! Yes, the ball does move a bit faster than the other tables, the gravity is higher and the flipper power is increased from its typical value.
Both the right 7-bank and inline targets on the left are supposed to be difficult shots, with the inline multiplier targets being especially high risk (but very high reward).

Here's my suggestion for high score strategy—focus first on clearing the 7-bank and 8-ball target to clear the rack. The bonus points from each rack completion carry over onto subsequent balls, so clearing racks early in the game will build a massive bonus. Only focus on the inline multiplier targets after completing a rack so that you can maximize the bonus collect behind the 8-ball target. The final red target is intentionally challenging and mostly serves as a high-value target to shoot for after clearing everything else on the table.

Appreciate all the feedback, I'm working on a few other tables in my spare time and will aim to make them a bit easier than this one!

**Comment by AlbeyAmakiir** [comment]
Unfortunately, no strategy can save a game where the ball can go from plunger to lost without touching flippers so often that one can lose two balls, even with the starting ball saver. :(

This does make me think that porting tables, at least accurately, might not be so appropriate for this game. I understand it's meant to be a difficult table, and you did tweak some rules, but it still feels unfair. And it also feels at odds with the rest of the game.

It's interesting to note, actually. I didn't realise how cohesive the other tables were until this one arrived.
- On this table, the fundamental physics are different, meaning experience with other tables in this game doesn't carry over nearly as much. If the other tables did not have this consistency, this wouldn't even register as an issue.
- Every other table has a way to make the table safer; that is, a way to restore outer lane ball saves.
- No other table allows the ball to go from plunger to lost without interaction with such frequency.

Some smaller things I would still consider issues, but I probably wouldn't have brought up if not for the above, since I know I'm quite a perfectionist:
- The visual language is different. For example, the other tables colour target sets green, rollovers cyan, bumpers blue or pink, etc. unless there's a specific reason not to. This helps communicate to the player (assuming they aren't colourblind, but that's a separate issue). This table breaks that language, and not always with apparent reason.
- The physical shapes are different, too. The walls have thickness, the flippers are at a very different angle, and so on. This has a small effect on expected ball behaviour, especially if the player doesn't realise.

What I would suggest, if porting a table from elsewhere, is to use the "accurate replication" as a base, then alter the design to match the rest of the game. As you said, you already changed some rules. So this would be an extention of that process. What I would do for this table is:
- Restore the physics of the rest of the game. A trick is, they don't actually need to be identical, but they need to be similar enough that the player *feels* like they are, and can use many of the expectations they have built up over the previous tables.
- Add outer lane ball save mechanics. This might be tricky, since that includes altering or adding targets to restore them, and that's additional design work beyond recreating mechanics that already exist.
- Alter the shapes and curves, and even the flipper separation, to lower the plunger-to-loss frequency. All this, but especially altering the flippers, may involve further adjusting the targets to better fit the new angles.
- Smaller adjustments for cohesion. Alter the lower section (flippers, outer lanes, and those triangle things I don't remember the name of) to match the shapes and angles of the rest of the tables. Adjust wall thickness. Colour match drop targets, rollover sets, bumpers. Make the bumpers outer colour darker. Sharpen some curves where other tables would not have curves.

That's a lot of work, I know. 😅
I really want to enjoy this table. It seems cool. I just think it needs more work in order to fit in *this* pinball game. How does all this sound?

**Comment by AlbeyAmakiir** [comment]
To be clear, I'm not saying it should be *easier*. But, if it's difficult, I'd want that difficulty to be less from bad luck and more from having my skill challenged.

**Comment by Gerbmouse** [comment]
I'd like to post also my 2 cents. First of all thanks for your time to create the table! Nonetheless I agree with all the comments above. I'd really like to play this table but also I'm not a bad player most of the time I loose the ball very quickly by bad luck (no chance to avoid it). Very often the ball goes from one bumper to the other than back and then into the lane. You state that you have made a fairly accurate replication. Is this behavior in the original table the same? Changing the angle of the bumpers slightly or rising or lowering the force of the bumpers may change this behavior totally. Maybe a first step. I don't think it needs extra gates.
